Unable to build 10.5.0 release in docker

just checked out the latest commits and tried to build docker image as usual:
```
Sending build context to Docker daemon  9.053MB
Step 1/19 : ARG DOTNET_VERSION=2.2
Step 2/19 : ARG FFMPEG_VERSION=latest
Step 3/19 : FROM node:alpine as web-builder
 ---> 4c6406de22fd
Step 4/19 : ARG JELLYFIN_WEB_VERSION=v10.5.0
 ---> Using cache
 ---> eab784468c80
Step 5/19 : RUN apk add curl  && curl -L https://github.com/jellyfin/jellyfin-web/archive/${JELLYFIN_WEB_VERSION}.tar.gz | tar zxf -  && cd jellyfin-web-*  && yarn install  && yarn build  && mv dist /dist
 ---> Running in 9bea1e715ce7
fetch http://dl-cdn.alpinelinux.org/alpine/v3.9/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.9/community/x86_64/APKINDEX.tar.gz
(1/5) Installing ca-certificates (20190108-r0)
(2/5) Installing nghttp2-libs (1.35.1-r1)
(3/5) Installing libssh2 (1.9.0-r0)
(4/5) Installing libcurl (7.64.0-r3)
(5/5) Installing curl (7.64.0-r3)
Executing busybox-1.29.3-r10.trigger
Executing ca-certificates-20190108-r0.trigger
OK: 9 MiB in 21 packages
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   130    0   130    0     0    580      0 --:--:-- --:--:-- --:--:--   577
100    15  100    15    0     0     26  tar: invalid magic
    0 --:--:-- --:--:-- --:--:--    26
tar: short read
The command '/bin/sh -c apk add curl  && curl -L https://github.com/jellyfin/jellyfin-web/archive/${JELLYFIN_WEB_VERSION}.tar.gz | tar zxf -  && cd jellyfin-web-*  && yarn install  && yarn build  && mv dist /dist' returned a non-zero code: 1
```

thanks for your quick replies first, I deleted all docker images and redownloaded them, now a new errors occurs, which isnt segfault:

```
Step 6/19 : FROM mcr.microsoft.com/dotnet/core/sdk:${DOTNET_VERSION} as builder
3.0: Pulling from dotnet/core/sdk
c7b7d16361e0: Pull complete 
b7a128769df1: Pull complete 
1128949d0793: Pull complete 
667692510b70: Pull complete 
efefb401e9dc: Pull complete 
9deb21478e67: Pull complete 
460e7bb49802: Pull complete 
475935cb0b6d: Pull complete 
Digest: sha256:66f93fea229f812f496441baf0877dd97d9e0c7fb2af209a1b055bcf4bb3919c
Status: Downloaded newer image for mcr.microsoft.com/dotnet/core/sdk:3.0
 ---> 170a7f2ec51a
Step 7/19 : WORKDIR /repo
 ---> Running in 01c3ce0e1b56
Removing intermediate container 01c3ce0e1b56
 ---> 1a9638126231
Step 8/19 : COPY . .
 ---> 23828970ee69
Step 9/19 : ENV DOTNET_CLI_TELEMETRY_OPTOUT=1
 ---> Running in 115920a698c1
Removing intermediate container 115920a698c1
 ---> e3e0b33e8d80
Step 10/19 : RUN dotnet publish Jellyfin.Server --configuration Release --output="/jellyfin" --self-contained --runtime linux-x64 "-p:GenerateDocumentationFile=false;DebugSymbols=false;DebugType=none"
 ---> Running in 03ebcd4864d3
Microsoft (R) Build Engine version 16.3.0+0f4c62fea for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Restore completed in 1.72 sec for /repo/BDInfo/BDInfo.csproj.
  Restore completed in 1.72 sec for /repo/Emby.Drawing/Emby.Drawing.csproj.
  Restore completed in 1.72 sec for /repo/DvdLib/DvdLib.csproj.
  Restore completed in 17.97 ms for /repo/Emby.Naming/Emby.Naming.csproj.
  Restore completed in 24.02 ms for /repo/Emby.Notifications/Emby.Notifications.csproj.
  Restore completed in 13.93 ms for /repo/Emby.XmlTv/Emby.XmlTv/Emby.XmlTv.csproj.
  Restore completed in 1.9 sec for /repo/Emby.Dlna/Emby.Dlna.csproj.
  Restore completed in 554.73 ms for /repo/Emby.Photos/Emby.Photos.csproj.
  Restore completed in 38.46 ms for /repo/MediaBrowser.Api/MediaBrowser.Api.csproj.
  Restore completed in 28.45 ms for /repo/MediaBrowser.Common/MediaBrowser.Common.csproj.
  Restore completed in 26.04 ms for /repo/MediaBrowser.Controller/MediaBrowser.Controller.csproj.
  Restore completed in 31.96 ms for /repo/MediaBrowser.LocalMetadata/MediaBrowser.LocalMetadata.csproj.
  Restore completed in 1.07 sec for /repo/MediaBrowser.MediaEncoding/MediaBrowser.MediaEncoding.csproj.
  Restore completed in 26.49 ms for /repo/MediaBrowser.Model/MediaBrowser.Model.csproj.
  Restore completed in 2.13 sec for /repo/MediaBrowser.Providers/MediaBrowser.Providers.csproj.
  Restore completed in 25.35 ms for /repo/MediaBrowser.WebDashboard/MediaBrowser.WebDashboard.csproj.
  Restore completed in 3.95 sec for /repo/Emby.Server.Implementations/Emby.Server.Implementations.csproj.
  Restore completed in 23 ms for /repo/MediaBrowser.XbmcMetadata/MediaBrowser.XbmcMetadata.csproj.
  Restore completed in 25.33 ms for /repo/Mono.Nat/Mono.Nat.csproj.
  Restore completed in 31.48 ms for /repo/RSSDP/RSSDP.csproj.
  Restore completed in 4.87 sec for /repo/Jellyfin.Drawing.Skia/Jellyfin.Drawing.Skia.csproj.
  Restore completed in 11.68 sec for /repo/Jellyfin.Server/Jellyfin.Server.csproj.
  MediaBrowser.Model -> /repo/MediaBrowser.Model/bin/Release/netstandard2.0/MediaBrowser.Model.dll
  Emby.XmlTv -> /repo/Emby.XmlTv/Emby.XmlTv/bin/Release/netstandard2.0/Emby.XmlTv.dll
  BDInfo -> /repo/BDInfo/bin/Release/netstandard2.0/BDInfo.dll
  DvdLib -> /repo/DvdLib/bin/Release/netstandard2.0/DvdLib.dll
  MediaBrowser.Common -> /repo/MediaBrowser.Common/bin/Release/netstandard2.0/MediaBrowser.Common.dll
  Emby.Naming -> /repo/Emby.Naming/bin/Release/netstandard2.1/Emby.Naming.dll
Upnp/UpnpNatDevice.cs(99,41): warning CS0618: 'IHttpClient.SendAsync(HttpRequestOptions, string)' is obsolete: 'Use 'Task{HttpResponseInfo} SendAsync(HttpRequestOptions options, HttpMethod httpMethod);' instead' [/repo/Mono.Nat/Mono.Nat.csproj]
Upnp/UpnpNatDevice.cs(230,26): warning CS0618: 'IHttpClient.SendAsync(HttpRequestOptions, string)' is obsolete: 'Use 'Task{HttpResponseInfo} SendAsync(HttpRequestOptions options, HttpMethod httpMethod);' instead' [/repo/Mono.Nat/Mono.Nat.csproj]
  Mono.Nat -> /repo/Mono.Nat/bin/Release/netstandard2.0/Mono.Nat.dll
  MediaBrowser.Controller -> /repo/MediaBrowser.Controller/bin/Release/netstandard2.0/MediaBrowser.Controller.dll
  Emby.Notifications -> /repo/Emby.Notifications/bin/Release/netstandard2.0/Emby.Notifications.dll
  MediaBrowser.WebDashboard -> /repo/MediaBrowser.WebDashboard/bin/Release/netstandard2.0/MediaBrowser.WebDashboard.dll
  MediaBrowser.MediaEncoding -> /repo/MediaBrowser.MediaEncoding/bin/Release/netstandard2.0/MediaBrowser.MediaEncoding.dll
ItemUpdateService.cs(64,35): warning CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread. [/repo/MediaBrowser.Api/MediaBrowser.Api.csproj]
  Jellyfin.Drawing.Skia -> /repo/Jellyfin.Drawing.Skia/bin/Release/netstandard2.0/Jellyfin.Drawing.Skia.dll
  MediaBrowser.LocalMetadata -> /repo/MediaBrowser.LocalMetadata/bin/Release/netstandard2.0/MediaBrowser.LocalMetadata.dll
Music/AudioDbAlbumProvider.cs(158,45): warning CS0618: 'IHttpClient.SendAsync(HttpRequestOptions, string)' is obsolete: 'Use 'Task{HttpResponseInfo} SendAsync(HttpRequestOptions options, HttpMethod httpMethod);' instead' [/repo/MediaBrowser.Providers/MediaBrowser.Providers.csproj]
Music/AudioDbArtistProvider.cs(144,45): warning CS0618: 'IHttpClient.SendAsync(HttpRequestOptions, string)' is obsolete: 'Use 'Task{HttpResponseInfo} SendAsync(HttpRequestOptions options, HttpMethod httpMethod);' instead' [/repo/MediaBrowser.Providers/MediaBrowser.Providers.csproj]
Music/MusicBrainzAlbumProvider.cs(764,34): warning CS0618: 'IHttpClient.SendAsync(HttpRequestOptions, string)' is obsolete: 'Use 'Task{HttpResponseInfo} SendAsync(HttpRequestOptions options, HttpMethod httpMethod);' instead' [/repo/MediaBrowser.Providers/MediaBrowser.Providers.csproj]
Tmdb/Movies/TmdbMovieProvider.cs(426,26): warning CS0618: 'IHttpClient.SendAsync(HttpRequestOptions, string)' is obsolete: 'Use 'Task{HttpResponseInfo} SendAsync(HttpRequestOptions options, HttpMethod httpMethod);' instead' [/repo/MediaBrowser.Providers/MediaBrowser.Providers.csproj]
  Emby.Photos -> /repo/Emby.Photos/bin/Release/netstandard2.0/Emby.Photos.dll
  MediaBrowser.Api -> /repo/MediaBrowser.Api/bin/Release/netstandard2.0/MediaBrowser.Api.dll
  RSSDP -> /repo/RSSDP/bin/Release/netstandard2.0/RSSDP.dll
  MediaBrowser.Providers -> /repo/MediaBrowser.Providers/bin/Release/netstandard2.0/MediaBrowser.Providers.dll
  Emby.Drawing -> /repo/Emby.Drawing/bin/Release/netstandard2.0/Emby.Drawing.dll
  MediaBrowser.XbmcMetadata -> /repo/MediaBrowser.XbmcMetadata/bin/Release/netstandard2.0/MediaBrowser.XbmcMetadata.dll
Eventing/EventManager.cs(179,30): warning CS0618: 'IHttpClient.SendAsync(HttpRequestOptions, string)' is obsolete: 'Use 'Task{HttpResponseInfo} SendAsync(HttpRequestOptions options, HttpMethod httpMethod);' instead' [/repo/Emby.Dlna/Emby.Dlna.csproj]
PlayTo/SsdpHttpClient.cs(93,26): warning CS0618: 'IHttpClient.SendAsync(HttpRequestOptions, string)' is obsolete: 'Use 'Task{HttpResponseInfo} SendAsync(HttpRequestOptions options, HttpMethod httpMethod);' instead' [/repo/Emby.Dlna/Emby.Dlna.csproj]
PlayTo/SsdpHttpClient.cs(113,41): warning CS0618: 'IHttpClient.SendAsync(HttpRequestOptions, string)' is obsolete: 'Use 'Task{HttpResponseInfo} SendAsync(HttpRequestOptions options, HttpMethod httpMethod);' instead' [/repo/Emby.Dlna/Emby.Dlna.csproj]
  Emby.Dlna -> /repo/Emby.Dlna/bin/Release/netstandard2.0/Emby.Dlna.dll
LiveTv/TunerHosts/SharedHttpStream.cs(66,34): warning CS0618: 'IHttpClient.SendAsync(HttpRequestOptions, string)' is obsolete: 'Use 'Task{HttpResponseInfo} SendAsync(HttpRequestOptions options, HttpMethod httpMethod);' instead' [/repo/Emby.Server.Implementations/Emby.Server.Implementations.csproj]
LiveTv/EmbyTV/DirectRecorder.cs(77,41): warning CS0618: 'IHttpClient.SendAsync(HttpRequestOptions, string)' is obsolete: 'Use 'Task{HttpResponseInfo} SendAsync(HttpRequestOptions options, HttpMethod httpMethod);' instead' [/repo/Emby.Server.Implementations/Emby.Server.Implementations.csproj]
LiveTv/Listings/SchedulesDirect.cs(666,30): warning CS0618: 'IHttpClient.SendAsync(HttpRequestOptions, string)' is obsolete: 'Use 'Task{HttpResponseInfo} SendAsync(HttpRequestOptions options, HttpMethod httpMethod);' instead' [/repo/Emby.Server.Implementations/Emby.Server.Implementations.csproj]
LiveTv/Listings/SchedulesDirect.cs(741,26): warning CS0618: 'IHttpClient.SendAsync(HttpRequestOptions, string)' is obsolete: 'Use 'Task{HttpResponseInfo} SendAsync(HttpRequestOptions options, HttpMethod httpMethod);' instead' [/repo/Emby.Server.Implementations/Emby.Server.Implementations.csproj]
  Emby.Server.Implementations -> /repo/Emby.Server.Implementations/bin/Release/netstandard2.1/Emby.Server.Implementations.dll
  Jellyfin.Server -> /repo/Jellyfin.Server/bin/Release/netcoreapp3.0/linux-x64/jellyfin.dll
  Jellyfin.Server -> /jellyfin/
Removing intermediate container 03ebcd4864d3
 ---> ebdfe46cec7f
Step 11/19 : FROM jellyfin/ffmpeg:${FFMPEG_VERSION} as ffmpeg
latest: Pulling from jellyfin/ffmpeg
5667fdb72017: Pull complete 
d83811f270d5: Pull complete 
ee671aafb583: Pull complete 
7fc152dfb3a6: Pull complete 
346fbe28cdae: Pull complete 
23d2c239732a: Pull complete 
e5ef79515baa: Pull complete 
d87927937b53: Pull complete 
Digest: sha256:c7e1c21e6bef44d20685c6b4c4812eb1473f42720fa0f29ac74f61b0058244ae
Status: Downloaded newer image for jellyfin/ffmpeg:latest
 ---> 76d3885fbbf3
Step 12/19 : FROM mcr.microsoft.com/dotnet/core/runtime:${DOTNET_VERSION}
3.0: Pulling from dotnet/core/runtime
8d691f585fa8: Already exists 
a9e6ceefc5a0: Already exists 
68f6aee67b91: Already exists 
7a6b64767b0b: Already exists 
Digest: sha256:e10cb4653af29abea2136c3633a3c904d0a6abc2c9720f221d68b116828d35f9
Status: Downloaded newer image for mcr.microsoft.com/dotnet/core/runtime:3.0
 ---> b525e32f0752
Step 13/19 : COPY --from=ffmpeg / /
 ---> fff1dbf2f246
Step 14/19 : COPY --from=builder /jellyfin /jellyfin
 ---> e33320c25d96
Step 15/19 : COPY --from=web-builder /dist /jellyfin/jellyfin-web
 ---> 72353e0459fb
Step 16/19 : RUN apt-get update  && apt-get install --no-install-recommends --no-install-suggests -y    libfontconfig1 mesa-va-drivers  && apt-get clean autoclean  && apt-get autoremove  && rm -rf /var/lib/apt/lists/*  && mkdir -p /cache /config /media  && chmod 777 /cache /config /media
 ---> Running in a34594bfe815
Get:1 http://archive.ubuntu.com/ubuntu bionic InRelease [242 kB]
Get:2 http://archive.ubuntu.com/ubuntu bionic-updates InRelease [88.7 kB]
Get:3 http://security.ubuntu.com/ubuntu bionic-security InRelease [88.7 kB]
Get:4 http://archive.ubuntu.com/ubuntu bionic-backports InRelease [74.6 kB]
Get:5 http://archive.ubuntu.com/ubuntu bionic/multiverse amd64 Packages [186 kB]
Get:6 http://archive.ubuntu.com/ubuntu bionic/universe amd64 Packages [11.3 MB]
Get:7 http://archive.ubuntu.com/ubuntu bionic/main amd64 Packages [1344 kB]
Get:8 http://security.ubuntu.com/ubuntu bionic-security/universe amd64 Packages [777 kB]
Get:9 http://archive.ubuntu.com/ubuntu bionic/restricted amd64 Packages [13.5 kB]
Get:10 http://archive.ubuntu.com/ubuntu bionic-updates/restricted amd64 Packages [21.9 kB]
Get:11 http://archive.ubuntu.com/ubuntu bionic-updates/multiverse amd64 Packages [8734 B]
Get:12 http://archive.ubuntu.com/ubuntu bionic-updates/universe amd64 Packages [1295 kB]
Get:13 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 Packages [977 kB]
Get:14 http://archive.ubuntu.com/ubuntu bionic-backports/universe amd64 Packages [4227 B]
Get:15 http://archive.ubuntu.com/ubuntu bionic-backports/main amd64 Packages [2496 B]
Get:16 http://security.ubuntu.com/ubuntu bionic-security/restricted amd64 Packages [11.3 kB]
Get:17 http://security.ubuntu.com/ubuntu bionic-security/multiverse amd64 Packages [5665 B]
Get:18 http://security.ubuntu.com/ubuntu bionic-security/main amd64 Packages [682 kB]
Fetched 17.2 MB in 2s (7029 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
The following additional packages will be installed:
  fontconfig-config fonts-dejavu-core libbsd0 libdrm-amdgpu1 libdrm-nouveau2
  libdrm-radeon1 libedit2 libelf1 libfreetype6 libllvm8 libpng16-16
  libx11-xcb1 libxau6 libxcb-dri2-0 libxcb-dri3-0 libxcb-present0 libxcb-sync1
  libxcb-xfixes0 libxcb1 libxdmcp6 libxshmfence1 multiarch-support ucf
The following NEW packages will be installed:
  fontconfig-config fonts-dejavu-core libbsd0 libdrm-amdgpu1 libdrm-nouveau2
  libdrm-radeon1 libedit2 libelf1 libfontconfig1 libfreetype6 libllvm8
  libpng16-16 libx11-xcb1 libxau6 libxcb-dri2-0 libxcb-dri3-0 libxcb-present0
  libxcb-sync1 libxcb-xfixes0 libxcb1 libxdmcp6 libxshmfence1 mesa-va-drivers
  multiarch-support ucf
0 upgraded, 25 newly installed, 0 to remove and 9 not upgraded.
Need to get 17.7 MB of archives.
After this operation, 89.1 MB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu bionic/main amd64 multiarch-support amd64 2.27-3ubuntu1 [6916 B]
Get:2 http://archive.ubuntu.com/ubuntu bionic/main amd64 libxau6 amd64 1:1.0.8-1 [8376 B]
Get:3 http://archive.ubuntu.com/ubuntu bionic/main amd64 libbsd0 amd64 0.8.7-1 [41.5 kB]
Get:4 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libelf1 amd64 0.170-0.4ubuntu0.1 [44.8 kB]
Get:5 http://archive.ubuntu.com/ubuntu bionic/main amd64 ucf all 3.0038 [50.5 kB]
Get:6 http://archive.ubuntu.com/ubuntu bionic/main amd64 libedit2 amd64 3.1-20170329-1 [76.9 kB]
Get:7 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpng16-16 amd64 1.6.34-1ubuntu0.18.04.2 [176 kB]
Get:8 http://archive.ubuntu.com/ubuntu bionic/main amd64 libxdmcp6 amd64 1:1.1.2-3 [10.7 kB]
Get:9 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libxcb1 amd64 1.13-2~ubuntu18.04 [45.5 kB]
Get:10 http://archive.ubuntu.com/ubuntu bionic/main amd64 fonts-dejavu-core all 2.37-1 [1041 kB]
Get:11 http://archive.ubuntu.com/ubuntu bionic/main amd64 fontconfig-config all 2.12.6-0ubuntu2 [55.8 kB]
Get:12 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libdrm-amdgpu1 amd64 2.4.97-1ubuntu1~18.04.1 [17.7 kB]
Get:13 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libdrm-nouveau2 amd64 2.4.97-1ubuntu1~18.04.1 [16.5 kB]
Get:14 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libdrm-radeon1 amd64 2.4.97-1ubuntu1~18.04.1 [21.7 kB]
Get:15 http://archive.ubuntu.com/ubuntu bionic/main amd64 libfreetype6 amd64 2.8.1-2ubuntu2 [335 kB]
Get:16 http://archive.ubuntu.com/ubuntu bionic/main amd64 libfontconfig1 amd64 2.12.6-0ubuntu2 [137 kB]
Get:17 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libllvm8 amd64 1:8-3~ubuntu18.04.1 [13.6 MB]
Get:18 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libx11-xcb1 amd64 2:1.6.4-3ubuntu0.2 [9376 B]
Get:19 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libxcb-dri2-0 amd64 1.13-2~ubuntu18.04 [6920 B]
Get:20 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libxcb-dri3-0 amd64 1.13-2~ubuntu18.04 [6568 B]
Get:21 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libxcb-present0 amd64 1.13-2~ubuntu18.04 [5552 B]
Get:22 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libxcb-sync1 amd64 1.13-2~ubuntu18.04 [8808 B]
Get:23 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libxcb-xfixes0 amd64 1.13-2~ubuntu18.04 [9352 B]
Get:24 http://archive.ubuntu.com/ubuntu bionic/main amd64 libxshmfence1 amd64 1.3-1 [5028 B]
Get:25 http://archive.ubuntu.com/ubuntu bionic-updates/universe amd64 mesa-va-drivers amd64 19.0.8-0ubuntu0~18.04.3 [1957 kB]
debconf: delaying package configuration, since apt-utils is not installed
Fetched 17.7 MB in 2s (8803 kB/s)
Selecting previously unselected package multiarch-support.
(Reading database ... 4422 files and directories currently installed.)
Preparing to unpack .../multiarch-support_2.27-3ubuntu1_amd64.deb ...
Unpacking multiarch-support (2.27-3ubuntu1) ...
Setting up multiarch-support (2.27-3ubuntu1) ...
Selecting previously unselected package libxau6:amd64.
(Reading database ... 4425 files and directories currently installed.)
Preparing to unpack .../00-libxau6_1%3a1.0.8-1_amd64.deb ...
Unpacking libxau6:amd64 (1:1.0.8-1) ...
Selecting previously unselected package libbsd0:amd64.
Preparing to unpack .../01-libbsd0_0.8.7-1_amd64.deb ...
Unpacking libbsd0:amd64 (0.8.7-1) ...
Selecting previously unselected package libelf1:amd64.
Preparing to unpack .../02-libelf1_0.170-0.4ubuntu0.1_amd64.deb ...
Unpacking libelf1:amd64 (0.170-0.4ubuntu0.1) ...
Selecting previously unselected package ucf.
Preparing to unpack .../03-ucf_3.0038_all.deb ...
Moving old data out of the way
Unpacking ucf (3.0038) ...
Selecting previously unselected package libedit2:amd64.
Preparing to unpack .../04-libedit2_3.1-20170329-1_amd64.deb ...
Unpacking libedit2:amd64 (3.1-20170329-1) ...
Selecting previously unselected package libpng16-16:amd64.
Preparing to unpack .../05-libpng16-16_1.6.34-1ubuntu0.18.04.2_amd64.deb ...
Unpacking libpng16-16:amd64 (1.6.34-1ubuntu0.18.04.2) ...
Selecting previously unselected package libxdmcp6:amd64.
Preparing to unpack .../06-libxdmcp6_1%3a1.1.2-3_amd64.deb ...
Unpacking libxdmcp6:amd64 (1:1.1.2-3) ...
Selecting previously unselected package libxcb1:amd64.
Preparing to unpack .../07-libxcb1_1.13-2~ubuntu18.04_amd64.deb ...
Unpacking libxcb1:amd64 (1.13-2~ubuntu18.04) ...
Selecting previously unselected package fonts-dejavu-core.
Preparing to unpack .../08-fonts-dejavu-core_2.37-1_all.deb ...
Unpacking fonts-dejavu-core (2.37-1) ...
Selecting previously unselected package fontconfig-config.
Preparing to unpack .../09-fontconfig-config_2.12.6-0ubuntu2_all.deb ...
Unpacking fontconfig-config (2.12.6-0ubuntu2) ...
Selecting previously unselected package libdrm-amdgpu1:amd64.
Preparing to unpack .../10-libdrm-amdgpu1_2.4.97-1ubuntu1~18.04.1_amd64.deb ...
Unpacking libdrm-amdgpu1:amd64 (2.4.97-1ubuntu1~18.04.1) ...
Selecting previously unselected package libdrm-nouveau2:amd64.
Preparing to unpack .../11-libdrm-nouveau2_2.4.97-1ubuntu1~18.04.1_amd64.deb ...
Unpacking libdrm-nouveau2:amd64 (2.4.97-1ubuntu1~18.04.1) ...
Selecting previously unselected package libdrm-radeon1:amd64.
Preparing to unpack .../12-libdrm-radeon1_2.4.97-1ubuntu1~18.04.1_amd64.deb ...
Unpacking libdrm-radeon1:amd64 (2.4.97-1ubuntu1~18.04.1) ...
Selecting previously unselected package libfreetype6:amd64.
Preparing to unpack .../13-libfreetype6_2.8.1-2ubuntu2_amd64.deb ...
Unpacking libfreetype6:amd64 (2.8.1-2ubuntu2) ...
Selecting previously unselected package libfontconfig1:amd64.
Preparing to unpack .../14-libfontconfig1_2.12.6-0ubuntu2_amd64.deb ...
Unpacking libfontconfig1:amd64 (2.12.6-0ubuntu2) ...
Selecting previously unselected package libllvm8:amd64.
Preparing to unpack .../15-libllvm8_1%3a8-3~ubuntu18.04.1_amd64.deb ...
Unpacking libllvm8:amd64 (1:8-3~ubuntu18.04.1) ...
Selecting previously unselected package libx11-xcb1:amd64.
Preparing to unpack .../16-libx11-xcb1_2%3a1.6.4-3ubuntu0.2_amd64.deb ...
Unpacking libx11-xcb1:amd64 (2:1.6.4-3ubuntu0.2) ...
Selecting previously unselected package libxcb-dri2-0:amd64.
Preparing to unpack .../17-libxcb-dri2-0_1.13-2~ubuntu18.04_amd64.deb ...
Unpacking libxcb-dri2-0:amd64 (1.13-2~ubuntu18.04) ...
Selecting previously unselected package libxcb-dri3-0:amd64.
Preparing to unpack .../18-libxcb-dri3-0_1.13-2~ubuntu18.04_amd64.deb ...
Unpacking libxcb-dri3-0:amd64 (1.13-2~ubuntu18.04) ...
Selecting previously unselected package libxcb-present0:amd64.
Preparing to unpack .../19-libxcb-present0_1.13-2~ubuntu18.04_amd64.deb ...
Unpacking libxcb-present0:amd64 (1.13-2~ubuntu18.04) ...
Selecting previously unselected package libxcb-sync1:amd64.
Preparing to unpack .../20-libxcb-sync1_1.13-2~ubuntu18.04_amd64.deb ...
Unpacking libxcb-sync1:amd64 (1.13-2~ubuntu18.04) ...
Selecting previously unselected package libxcb-xfixes0:amd64.
Preparing to unpack .../21-libxcb-xfixes0_1.13-2~ubuntu18.04_amd64.deb ...
Unpacking libxcb-xfixes0:amd64 (1.13-2~ubuntu18.04) ...
Selecting previously unselected package libxshmfence1:amd64.
Preparing to unpack .../22-libxshmfence1_1.3-1_amd64.deb ...
Unpacking libxshmfence1:amd64 (1.3-1) ...
Selecting previously unselected package mesa-va-drivers:amd64.
Preparing to unpack .../23-mesa-va-drivers_19.0.8-0ubuntu0~18.04.3_amd64.deb ...
Unpacking mesa-va-drivers:amd64 (19.0.8-0ubuntu0~18.04.3) ...
Setting up libedit2:amd64 (3.1-20170329-1) ...
Setting up libllvm8:amd64 (1:8-3~ubuntu18.04.1) ...
Setting up libdrm-amdgpu1:amd64 (2.4.97-1ubuntu1~18.04.1) ...
Setting up libpng16-16:amd64 (1.6.34-1ubuntu0.18.04.2) ...
Setting up fonts-dejavu-core (2.37-1) ...
Setting up libelf1:amd64 (0.170-0.4ubuntu0.1) ...
Setting up libxshmfence1:amd64 (1.3-1) ...
Setting up libbsd0:amd64 (0.8.7-1) ...
Setting up ucf (3.0038) ...
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
debconf: unable to initialize frontend: Readline
debconf: (Can't locate Term/ReadLine.pm in @INC (you may need to install the Term::ReadLine module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.26.1 /usr/local/share/perl/5.26.1 /usr/lib/x86_64-linux-gnu/perl5/5.26 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.26 /usr/share/perl/5.26 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at /usr/share/perl5/Debconf/FrontEnd/Readline.pm line 7.)
debconf: falling back to frontend: Teletype
Setting up libfreetype6:amd64 (2.8.1-2ubuntu2) ...
Setting up libx11-xcb1:amd64 (2:1.6.4-3ubuntu0.2) ...
Setting up libdrm-radeon1:amd64 (2.4.97-1ubuntu1~18.04.1) ...
Setting up libxdmcp6:amd64 (1:1.1.2-3) ...
Setting up libdrm-nouveau2:amd64 (2.4.97-1ubuntu1~18.04.1) ...
Setting up libxau6:amd64 (1:1.0.8-1) ...
Setting up fontconfig-config (2.12.6-0ubuntu2) ...
Setting up libxcb1:amd64 (1.13-2~ubuntu18.04) ...
Setting up libxcb-present0:amd64 (1.13-2~ubuntu18.04) ...
Setting up libfontconfig1:amd64 (2.12.6-0ubuntu2) ...
Setting up libxcb-dri2-0:amd64 (1.13-2~ubuntu18.04) ...
Setting up libxcb-dri3-0:amd64 (1.13-2~ubuntu18.04) ...
Setting up libxcb-xfixes0:amd64 (1.13-2~ubuntu18.04) ...
Setting up libxcb-sync1:amd64 (1.13-2~ubuntu18.04) ...
Setting up mesa-va-drivers:amd64 (19.0.8-0ubuntu0~18.04.3) ...
Processing triggers for libc-bin (2.27-3ubuntu1) ...
E: Problem executing scripts DPkg::Post-Invoke 'rm -f /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin || true'
E: Sub-process returned an error code
The command '/bin/sh -c apt-get update  && apt-get install --no-install-recommends --no-install-suggests -y    libfontconfig1 mesa-va-drivers  && apt-get clean autoclean  && apt-get autoremove  && rm -rf /var/lib/apt/lists/*  && mkdir -p /cache /config /media  && chmod 777 /cache /config /media' returned a non-zero code: 100
```

seems the ubuntu built-container is broken?

--- a/Dockerfile
+++ b/Dockerfile
@@ -17,8 +17,8 @@ ENV DOTNET_CLI_TELEMETRY_OPTOUT=1
 RUN dotnet publish Jellyfin.Server --configuration Release --output="/jellyfin" --self-contained --runtime linux-x64 "-p:GenerateDocumentationFile=false;DebugSymbols=false;DebugType=none"
 
 FROM jellyfin/ffmpeg:${FFMPEG_VERSION} as ffmpeg
+FROM debian:stretch-slim
 
-FROM mcr.microsoft.com/dotnet/core/runtime:${DOTNET_VERSION}
 COPY --from=ffmpeg /opt/ffmpeg /opt/ffmpeg
 COPY --from=builder /jellyfin /jellyfin
 COPY --from=web-builder /dist /jellyfin/jellyfin-web
@@ -38,9 +38,11 @@ RUN apt-get update \
  && ln -s /opt/ffmpeg/bin/ffmpeg /usr/local/bin \
  && ln -s /opt/ffmpeg/bin/ffprobe /usr/local/bin
 
+ENV DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=1
+
 EXPOSE 8096
 VOLUME /cache /config /media
-ENTRYPOINT dotnet /jellyfin/jellyfin.dll \
+ENTRYPOINT ./jellyfin/jellyfin \
     --datadir /config \
     --cachedir /cache \
     --ffmpeg /usr/local/bin/ffmpeg

--- a/Dockerfile.arm
+++ b/Dockerfile.arm
@@ -24,7 +24,7 @@ RUN dotnet publish Jellyfin.Server --configuration Release --output="/jellyfin" 
 
 
 FROM multiarch/qemu-user-static:x86_64-arm as qemu
-FROM mcr.microsoft.com/dotnet/core/runtime:${DOTNET_VERSION}-stretch-slim-arm32v7
+FROM debian:stretch-slim-arm32v7
 COPY --from=qemu /usr/bin/qemu-arm-static /usr/bin
 RUN apt-get update \
  && apt-get install --no-install-recommends --no-install-suggests -y ffmpeg \
@@ -34,9 +34,11 @@ RUN apt-get update \
 COPY --from=builder /jellyfin /jellyfin
 COPY --from=web-builder /dist /jellyfin/jellyfin-web
 
+ENV DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=1
+
 EXPOSE 8096
 VOLUME /cache /config /media
-ENTRYPOINT dotnet /jellyfin/jellyfin.dll \
+ENTRYPOINT ./jellyfin/jellyfin \
     --datadir /config \
     --cachedir /cache \
     --ffmpeg /usr/bin/ffmpeg

--- a/Dockerfile.arm64
+++ b/Dockerfile.arm64
@@ -24,7 +24,7 @@ RUN dotnet publish Jellyfin.Server --configuration Release --output="/jellyfin" 
 
 
 FROM multiarch/qemu-user-static:x86_64-aarch64 as qemu
-FROM mcr.microsoft.com/dotnet/core/runtime:${DOTNET_VERSION}-stretch-slim-arm64v8
+FROM debian:stretch-slim-arm64v8
 COPY --from=qemu /usr/bin/qemu-aarch64-static /usr/bin
 RUN apt-get update \
  && apt-get install --no-install-recommends --no-install-suggests -y ffmpeg \
@@ -34,9 +34,11 @@ RUN apt-get update \
 COPY --from=builder /jellyfin /jellyfin
 COPY --from=web-builder /dist /jellyfin/jellyfin-web
 
+ENV DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=1
+
 EXPOSE 8096
 VOLUME /cache /config /media
-ENTRYPOINT dotnet /jellyfin/jellyfin.dll \
+ENTRYPOINT ./jellyfin/jellyfin \
     --datadir /config \
     --cachedir /cache \
     --ffmpeg /usr/bin/ffmpeg

